doc/releases/changelog-dev.md

```diff
@@ -272,4 +293,6 @@ Pablo Antonio Moreno Casares,
 Lee J. O'Riordan,
 Mudit Pandey,
 Alex Preciado,
-Matthew Silverman.
+Matthew Silverman,
+Diksha Dhawan.
```

Comment:
Hi @ddhawan11, not a blocker for merging but we go in alphabetical order based on last name, so you can move to below Isaac.